cfme/tests/test_rest.py

```diff
@@ -374,13 +374,446 @@ def test_vm_add_lifecycle_event(request, setup_a_provider, rest_api, vm, from_de
 }
 
 
+@pytest.fixture(scope="module")
+def users_data():
+    name = fauxfactory.gen_alphanumeric()
+    result = [{
+        "name": "name_{}".format(name),
+        "userid": "userid_{}".format(name),
+        "email": "{}@local.com".format(name),
+    } for _ in range(1, 5)]
```

Comment:
Both here and in the cases below, you are generating the same data using the range() func - I only skimmed through the PR but if you plan to use the data to add multiple users at once, this won't (shouldn't) work. The user IDs will have to differ. And I would expect a password to be necessary as well.